packages/hop-node/src/db/TransferRootsDb.ts

```diff
@@ -213,6 +213,31 @@ class TransferRootsDb extends BaseDb {
       )
     })
   }
+
+  async getUnsetCommittedTransferRoots (filter: Partial<TransferRoot>): Promise<TransferRoot[]> {
```

Comment:
That makes sense, will updated to use `getUnbondedTransferRoots()`.

The reason that currently getUnbondedTransferRoots doesn't take in source/destination is because there's only one instance of BondTransferRootWatcher (only L1) so it's not concerned about filtering